openest/test/curves/test_linextrap.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+import numpy as np
+
+from openest.models.curve import ZeroInterceptPolynomialCurve, CoefficientsCurve
+from openest.curves.linextrap import *
```

Comment:
Mentioned in earlier comment but I'd change this wildcard import. They makes it unclear what's actually in the namespace, what's being tested/used as dependency, etc.